conans/cli/commands/__init__.py

```diff
@@ -32,3 +33,35 @@ def default(self, o):
 def json_formatter(data):
     myjson = json.dumps(data, indent=4, cls=ConanJSONEncoder)
     return myjson
+
+
+def add_log_level_args(subparser):
+    subparser.add_argument("-v", "--output-level",  default="status", nargs='?',
+                           help="Level of detail of the output")
```

Comment:
May we add some info of what are the valid output levels and order them from the least verbose to the most verbose? There's no help even if you use a non-valid output level.

Reply:
done